.github/workflows/docker-publish.yml

```
@@ -67,3 +68,28 @@ jobs:
         run: |
           docker push ${{ secrets.DOCKER_USERNAME }}/playus-user-service:${{ env.BASE_TAG }}
           docker push ${{ secrets.DOCKER_USERNAME }}/playus-user-service:${{ env.BUILD_VERSION }}
+
+      - name: Update patch-image.yaml
+        if: success()
+
+        run: |
+          git clone https://x-access-token:${{ secrets.K8S_REPO_TOKEN }}@github.com/KEA-ChunSam/PlayUs-k8s.git
+
+          BRANCH_NAME=${GITHUB_REF##*/}
+          if [[ "$BRANCH_NAME" == "main" ]]; then 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)